dojoWordle/components/LetterBox.tsx/LetterBox.style.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "@emotion/native";
+import { LetterBoxProps } from "./LetterBox";
+
+const KEY_HEIGHT = 55;
```

Comment:
```suggestion
const KEY_SIZE = 55;
```

Because it defines height AND width :)